release-notes/release-notes-v0.5.0.md

```diff
@@ -0,0 +1,433 @@
+commands check [the docs](https://bitcoin-s.org/docs/0.5.0/applications/server#server-endpoints).
+
+The `Cli` module is now published as a library, this should allow other projects to easily interact with a Bitcoin-S server.
+
+#### Commits
+
+2be2df12dbb Publish Cli as library (#2433)
+847981317a0 Add getblockheader cli command (#2424)
+f135322c094 Bump Fee Cli commands (#2415)
+a2b54eef308 Import Seed cli commands (#2376)
+ce9fbb0807a Analyze PSBT function (#2240)
+c08379b2361 Decode PSBT function (#2237)
+c662cf802d1 Make tx an argument for decoderawtransaction (#2155)
+8c4308e2c6a Make cli pretty print json (#2130)
+
+### Commons
+
+#### Commits
+
+### Core
+
+Fixed some bugs with PSBT parsing and adding witness scripts.
+
+Initial BIP 155 support for sending and receiving addrv2 messages.
+
+Initial BIP 325 support for interacting with Signet.
+
+Experimental TLV support for DLC messaging.
+These are subject to change and if your project is looking to use them
+it is recommended to use [nightly builds](https://bitcoin-s.org/docs/next/getting-started#nightly-builds) to stay up to date.
+
+#### Commits
+
+036d714563b Made TLV serialization and deserialization uniform under a succinct and expressive API (#2420)
+13c4b0d9556 Added length prefixes to contract_info and cet_signatures TLVs (#2419)
+719fab23b7a Remove CompatEither, it was needed for historical purposes to support… (#2394)
+378c51991b9 DLC Data Structures on Master Cleanup (#2375)
+49a281acaf0 Pulled down work from adaptor-dlc onto master (#2339)
+49871df0c35 Require CJDNS starts with 0xFC (#2356)
+3e7fade2240 Add latest ProtocolVersion (#2332)
+13443fd0d1e Implement BIP 155 addrv2 messages (#2321)
+b3d70f559a6 Fix TLV parsing for non-standard strings (#2312)
+0c03aa9c24d Add function to validate OracleAnnouncementV0TLV's signature (#2308)
+28c37ccf1fe trivial: Add a sha256 hash field on all TLV (#2300)
+b638c6837e6 Move english word list to be represented by a Vector in memory (#2287)
+18dfbed8c90 Add helper OracleEventTLVV0.maturation method (#2267)
+e4194220c1e Update Oracle TLVs (#2185)
+7178cb7136f Fixed P2SH(Segwit) bug and brought down DLC-used PSBT functionality (#2140)
+ddf39057fe0 Only Validate PSBT BIP143 vulnerability on signing (#2204)
+02c10fd89ed Pretty Fee Rate toStrings (#2210)
+c275a8c8aa6 Make tx bytes functions lazy vals (#2180)
+c5be81d5e21 Add address descriptors (#2176)
+2e1b4d04919 P2SHTxSigComponent constructor now detects witness data (#2169)
+7bbc89fb33f Optimize GetHeadersMessage.fromBytes (#2131)
+4ac93660d2c Add BIP45 Multisig Purpose (#2103)
+f69678d74bf Allow any HDCoinType (#2097)
+fa31609daca Enable AddrMessageTest (#2106)
+d158f4fcb08 Calculate HRP from network param instead of the inverse (#2079)
+c2315082d8c Create ExtPrivateKeyHardened (#2073)
+856f88f0ddb Initial SigNet support (#2057)
+
+### Crypto
+
+ECDSA Adaptor signatures have been added in a basic Java implementation. These should be used with caution as they are
```

Comment:
Link to this PR i think is the best place? https://github.com/discreetlogcontracts/dlcspecs/pull/114